// angular-imask/esm2020/directive.module.mjs
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { IMaskDirective } from './imask.directive';
import { IMaskFactory } from './imask-factory';
import { DefaultImaskFactory } from './default-imask-factory';
import * as i0 from "@angular/core";
export class IMaskDirectiveModule {
}
IMaskDirectiveModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "14.2.2", ngImport: i0, type: IMaskDirectiveModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IMaskDirectiveModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "14.0.0", version: "14.2.2", ngImport: i0, type: IMaskDirectiveModule, declarations: [IMaskDirective], imports: [CommonModule], exports: [IMaskDirective] });
IMaskDirectiveModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "14.2.2", ngImport: i0, type: IMaskDirectiveModule, providers: [{ provide: IMaskFactory, useClass: DefaultImaskFactory }], imports: [CommonModule] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "14.2.2", ngImport: i0, type: IMaskDirectiveModule, decorators: [{
            type: NgModule,
            args: [{
                    imports: [CommonModule],
                    declarations: [IMaskDirective],
                    providers: [{ provide: IMaskFactory, useClass: DefaultImaskFactory }],
                    exports: [IMaskDirective]
                }]
        }] });
