// angular-imask/esm2020/imask.module.mjs
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { IMaskPipe } from './imask.pipe';
import { IMaskDirectiveModule } from './directive.module';
import * as i0 from "@angular/core";
export class IMaskModule {
}
IMaskModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "14.2.2", ngImport: i0, type: IMaskModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IMaskModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "14.0.0", version: "14.2.2", ngImport: i0, type: IMaskModule, declarations: [IMaskPipe], imports: [CommonModule, IMaskDirectiveModule], exports: [IMaskPipe, IMaskDirectiveModule] });
IMaskModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "14.2.2", ngImport: i0, type: IMaskModule, imports: [CommonModule, IMaskDirectiveModule, IMaskDirectiveModule] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "14.2.2", ngImport: i0, type: IMaskModule, decorators: [{
            type: NgModule,
            args: [{
                    imports: [CommonModule, IMaskDirectiveModule],
                    declarations: [IMaskPipe],
                    exports: [IMaskPipe, IMaskDirectiveModule]
                }]
        }] });
